Source/GameEngine/Models/Die.cs

```diff
@@ -18,8 +18,9 @@ public Dice()
         public int RollDice()
```

Comment:
Why ain't this renamed to Die?

Reply:
I actually don't know. Thanks for noticing it!